### DISCUSSION TITLE
User unable to login into multiple jurisdictions

### DISCUSSION THREAD

**Discussion by TCGSourav** [egovernments/Digit-Core#198] [Bugs]
At PGR module, we can create user with multiple jurisdictions. But the same user cannot login into all those cities, declared in his profile.  
![P1](https://github.com/egovernments/Digit-Core/assets/142198762/aef436ba-0ab9-4c93-b24c-32baa3289b05)

**Comment by shashwat-egov** [comment]
Yes, the user can only login to the primary tenant by selecting it in the login screen. Once logged in, on the top bar, the user can change the city present in the dropdown based on the jurisdiction they have to take required actions.
